README.md

```diff
@@ -85,15 +85,27 @@ Castle.configure do |config|
   # We always blacklist Cookie and Authentication headers. If you use any other headers that
   # might contain sensitive information, you should blacklist them.
   config.blacklisted = ['HTTP-X-header']
+
+  # Castle needs the original IP of the client, not the IP of your proxy or load balancer.
+  # we try to fetch proper ip based on X-Forwarded-For, X-Client-Id or Remote-Addr headers in that order
+  # but sometimes proper ip may be stored in different header or order could be different.
+  # SDK can extract ip automatically in that case too, but you have to configure it ip_headers 
+  # which you would like to use in the first place
```

Comment:
> but you have to configure it ip_headers which you would like to use in the first place

can we rephrase that? ;)

Reply:
how about "but you must configure which ip_headers you would like to use"